### PR TITLE
FOGL-6962 Add new hint details

### PR DESCRIPTION
**Pull request by MarkRiddoch**
Signed-off-by: Mark Riddoch <mark@dianomic.com>

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -55,6 +55,18 @@ The following hint types are currently supported by |OMFNorth|:
...
+
+  - *LegacyType*: This defines the way OMF types are used and provides a mechanism to use types in the way the initial OMF plugin used them. Unless you have reason to force this older behaviour it is recomended to not use this hint. This hint is fully documented in the |OMFNorth| plugin page.
+
+  - *uom*: Define the unit of mesaure for the datapoint. This can only appear within datapoints and not for assets.
+
+  - *minimum*: Define the minimum value for a datapoint. This can only appear within datapoints and not for assets.
+
+  - *maximum*: Define the maximum value for a datapoint. This can only appear within datapoints and not for assets.
+
+  - *interpoaltion*: Define the interpolation value for a datapoint. This can only appear within datapoints and not for assets. Interpolation may be set to one of *continuous*, *discrete*, *stepwisecontinuousleading*, and *stepwisecontinuousfollowing*.
```
Comment (RVerhoeff) [inline docs/index.rst]: Typo: interpoaltion -> interpolation

File: docs/index.rst [hunk 2]
```diff
@@ -117,3 +129,24 @@ Datapoint hints can be combined with asset hints
...
+      "motor4" : {
+        "OMFHint"  : { "datapoint" : [
+                { "name" : "voltage:, "number" : "float32", "uom" : "volt" },
+                { "name" : "current:, "number" : "uint32", "uom" : "milliampere }
+                ]
+        }
+      }
+   }
+
+The example above attaches a number hint to both the volatage and current
```
Comment (RVerhoeff) [inline docs/index.rst]: I would split this into 3 shorter sentences: "The example above attaches a number hint to both the voltage and current
datapoints. To the current datapoint it assigns a unit of measure of milliamepere. The unit of measure for the voltage is set to be volts." There are two typos in this paragraph: volatage -> voltage, milliamepere -> milliampere.

**Review by RVerhoeff — CHANGES_REQUESTED** [review]
General comment: our "proper nouns" are all in lower-case: _datapoint_, _reading_ and _asset_. Should we change this to leading capital letters? I realize this applies to many of our documentation pages.

**Comment by MarkRiddoch** [comment]
> 

Probably, let's do it has a general cleanup sweep however rather than piecemeal

**Review by RVerhoeff — APPROVED** [review]
All good

**Pull request merged**